specification/validated-relationships-service-api.yaml

```diff
@@ -279,11 +279,17 @@ paths:
               examples:  
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
+
 components:
   schemas:
     QuestionnaireResponse:
       type: object
       description: A FHIR QuestionnaireResponse
+      required:
+        - status
```

Comment:
- make `item` required otherwise a valid response can contain no answers

Q: I understand why you've not made `subject` required, but might be better to do so to make the response more self documenting/declarative